index.php

```diff
@@ -36,10 +36,10 @@
 // Check PHP version not to have trouble
 // Need to be the very fist step before any include
 if (
-    version_compare(PHP_VERSION, '7.4.0', '<') ||
-    version_compare(PHP_VERSION, '8.2.0', '>=')
+    version_compare(PHP_VERSION, '7.4.0-dev', '<') ||
```

Comment:
why those `-dev`?

Reply:
we should have same versions ins both files

Reply:
`-dev` suffixes are added during comparison in requirement check, see https://github.com/glpi-project/glpi/blob/0046d6d6b5d7fc509ca139c62d4c2e65d2ce4d50/src/System/Requirement/PhpVersion.php#L79-L84

So I added `-dev` suffix here to do the exact same comparison.

Reply:
> we should have same versions ins both files

That does not explain this difference